language-service/language_service/service/vocab/cedict/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)